packages/components/app/styles/components/pagination.scss

```diff
@@ -31,22 +31,47 @@ $pagination-layout-breakpoint: 1000px;
     flex-wrap: wrap;
     justify-content: center;
   }
+
+  // Apply group layout styles to child components only when used inside pagination wrapper:
```

Comment:
Super nitpick: in the subsequent code, the order of declarations is: "info" / "nav" / "size-selector", we could follow the same order here